lib/webmachine/errors.rb

```diff
@@ -39,6 +39,6 @@ class InvalidResource < Error; end
 
   # Raised when the client has submitted an invalid request, e.g. in
   # the case where a request header is improperly formed. Raising this
-  # exception will result in a 400 response.
+  # error will result in a 400 response.
```

Comment:
Even though the exception has the name "Error", it's still an exception. 
I haven't heard "raise an error" much, i think "raise an exception" is more common and understood by all Ruby programmers (afaik).